### PR TITLE
Added 2024 SC nomination post

### PR DESCRIPTION
**Pull request by jdamerow**

### CODE REVIEW

File: content/blog/2024-10-31-sc-nominations.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title: DHTech Steering Committee Nominations 2024
+date: 2024-10-31T12:00:00
+author: DHTech Steering Committee
+tags:
+- WhatsHappening
+thumbnail: /images/vote.jpeg
+featureImage: /images/vote.jpeg
+featureImageAlt: photo of a crowd in an auditorium with many people raising their hands
+featureImageCap: Photo by <a href="https://unsplash.com/@theunsteady5?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Edwin Andrade</a> on <a href="https://unsplash.com/photos/4V1dC_eoCwg?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
+summary: This November we are planning to hold the first official Steering Committee elections. This blog post summarizes the key responsibilities and tasks of the Steering Committee.
```
Comment (ColeDCrawford) [inline content/blog/2024-10-31-sc-nominations.md:12]: Not the first one, right?

File: content/blog/2024-10-31-sc-nominations.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+url: 2024-sc-nominations
+---
+DHTech has been an ADHO Special Interest Group since 2021. For the upcoming two-year term (2025-2026), three seats on the Steering Committee need to be filled. Any member of DHTech is eligible to nominate themselves for one of the spots. To give people an idea of what it involves to be on the Steering Committee, this blog post will summarize the key responsibilities and tasks of the Steering Committee. We hope that this will encourage some people to nominate themselves as candidates.
+
+## What does the Steering Committee do?
+The DHTech Steering Committee supports the DHTech community by planning events, maintaining the community website and Slack workspace, and providing a contact for ADHO in regards to our Special Interest Group (SIG) status. For example, we have organized workshops and panels at ACH2021, DH2022, DH2023, and DH2024 as well as presented posters. We have organized virtual meetups for the DHTech community to discuss topics and present projects. We also maintain the DHTech website (dhtech.github.io) and mailing list, and complile a bi-monthly newsletter.
```
Comment (ColeDCrawford) [inline content/blog/2024-10-31-sc-nominations.md:18]: Link the site?

File: content/blog/2024-10-31-sc-nominations.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+If you have a few hours each month to give to DHTech and are a DHTech member (no matter when you joined), then yes, you are! There is no other requirement to be on the Steering Committee except a desire to support the community and help DHTech grow. 
+
+## What do I need to do to nominate myself?
+If you want to nominate yourself as a candidate for the DHTech Steering Committee, please fill out [this form](https://forms.gle/2RbYgXxRQKCbBN3m9). You will need to provide your name, affiliation, and a short statement (e.g. about who you are and why you want to be on the DHTech steering committee). 
+
+## What’s the nomination deadline?
+Nominations are open until Friday, November 15, 2024.
+
+## Can I get involved in other ways?
+Yes! There is the Code Review Working Group and the Education & Training Woring Group that both meet monthly if that is of interest to you. Or you can join our bi-monthly meetups. We also always welcome blog posts for our website. If you have specific ideas of things you would like to do, please post them on Slack or contact a member of the Steering Committee directly.
```
Comment (ColeDCrawford) [inline content/blog/2024-10-31-sc-nominations.md:35]: sp "Working Group"

File: content/blog/2024-10-31-sc-nominations.md (hunk 1 above)
Comment (jdamerow) [reply content/blog/2024-10-31-sc-nominations.md:12]: I keep forgetting to adjust the summaries 🤦‍♀️ One day...

**Pull request merged**